packages/playground/blueprints/public/blueprint-schema.json

```diff
@@ -5,6 +5,10 @@
 		"Blueprint": {
 			"type": "object",
 			"properties": {
+				"helloWorld": {
```

Comment:
Just to make sure – did you update this by hand? Or did you autogenerate this? There's an underdocumented nx command `nx run playground-blueprints:build:blueprint-schema` that turns TS types into this schema :-)

Reply:
No, manually. I ran it now. This is also a good thing for the documentation.